torch/quantization/ns/graph_passes.py

```diff
@@ -266,9 +311,12 @@ def create_a_shadows_b(
     def load_arg(a):
         return map_arg(a, lambda node: env_c[node.name])
 
-    nodes_to_instrument_b_to_a = {}
-    for match_name, (node_a, node_b) in matched_node_pairs.items():
-        nodes_to_instrument_b_to_a[node_b] = node_a
+    node_b_to_matched_subgraph_a = {}
```

Comment:
It is quite useful to insert shadow at multiple levels.  Assuming in a later PR we will support shadowing a sub-graph instead of a single node.

Reply:
sounds reasonable